### PR TITLE
Wdp 124

### PR DESCRIPTION
**Pull request by gonczor**
Templates created
image
image
https://kindergarten.stxnext.pl/secure/RapidBoard.jspa?rapidView=17&projectKey=WDP&view=detail&selectedIssue=WDP-124

### CODE REVIEW

File: src/presence_analyzer/views.py [hunk 1]
```diff
@@ -99,3 +101,18 @@ def presence_start_end_view(user_id):
...
+
+
+@app.route('/presence_weekday/')
+def presence_weekday_view():
+    return render_template('presence_weekday.html')
+
+
+@app.route('/presence_start_end/')
+def presence_start_end_view():
+    return render_template('presence_start_end.html')
```
Comment (piotrgrundas) [inline src/presence_analyzer/views.py]: 1 handler for all, param sets tpl

File: src/presence_analyzer/templates/main.html [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            <li><a href="{{ url_for('presence_start_end_view') }}">Presence start-end</a></li>
+        {% elif request.path == '/presence_start_end/' %}
+            <li><a href="{{ url_for('presence_weekday_view') }}">Presence by weekday</a></li>
+            <li><a href="{{ url_for('mean_time_weekday_view') }}">Presence mean time</a></li>
+            <li id="selected"><a href="{{ url_for('presence_start_end_view') }}">Presence start-end</a></li>
+        {% elif request.path == '/presence_weekday/' %}
+            <li id="selected"><a href="{{ url_for('presence_weekday_view') }}">Presence by weekday</a></li>
+            <li><a href="{{ url_for('mean_time_weekday_view') }}">Presence mean time</a></li>
+            <li><a href="{{ url_for('presence_start_end_view') }}">Presence start-end</a></li>
+        {% endif %}
```
Comment (piotrgrundas) [inline src/presence_analyzer/templates/main.html]: Do it in a loop

File: src/presence_analyzer/static/js/presence_weekday.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                    var options = {};
+                    chart_div.show();
+                    loading.hide();
+                    var chart = new google.visualization.PieChart(chart_div[0]);
+                    chart.draw(data, options);
+                });
+            }
+        });
+    });
+})(jQuery);
```
Comment (piotrgrundas) [inline src/presence_analyzer/static/js/presence_weekday.js]: This shouldn't be here, configure ur editor
image

File: src/presence_analyzer/static/js/presence_weekday.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+google.load("visualization", "1", {packages:["corechart"], 'language': 'en'});
```
Comment (piotrgrundas) [inline src/presence_analyzer/static/js/presence_weekday.js]: Once u declare in tpl above, then here. Make up ur mind.

**Review by piotrgrundas — CHANGES_REQUESTED** [review]

**Comment by piotrgrundas** [comment]
Fix commits

File: src/presence_analyzer/views.py (hunk 1 above)
Comment (gonczor) [reply src/presence_analyzer/views.py]: Done

File: src/presence_analyzer/static/js/presence_weekday.js (hunk 3 above)
Comment (gonczor) [reply src/presence_analyzer/static/js/presence_weekday.js]: Done

File: src/presence_analyzer/templates/main.html (hunk 2 above)
Comment (gonczor) [reply src/presence_analyzer/templates/main.html]: Done

**Pull request closed without merging**